src/render.js

```diff
@@ -107,3 +110,28 @@ function debug(
   debugImpl.shallow = (message) => debugShallow(instance, message);
   return debugImpl;
 }
+
+function appendFindAllTrap(renderer: ReactTestRenderer) {
+  return new Proxy(renderer.root, {
+    get(target, prop) {
+      const isFindAllProp = prop === 'findAll';
+      const newFindAll = (fn, node = target) => {
+        if (node.props.accessibilityElementsHidden) {
+          return [];
+        }
+
+        const initial = fn(node) ? [node] : [];
+
+        return node.children.reduce((result, child) => {
+          if (typeof child === 'string') {
```

Comment:
Seems like we need also to handle `number` type here as well (maybe also `boolean`). Please create tests with `<Text>{1}</Text>`, `{false}`, `{null}`, `{undefined}` and see if it crashes.